deploy/helm/tracee/Chart.yaml

```diff
@@ -18,7 +18,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: "0.19.0"
+version: "0.19.1"
```

Comment:
let's keep the same version, as we are releasing the chart along with tracee, the 0.20.0 release will be soon, and we can bump it before releasing